File: f1-api/model/calendar.go
package model

// Event :
type Event struct {
	Day     string `json:"Day,omitempty"`
	Circuit string `json:"Circuit,omitempty"`
}

// Calendar :
type Calendar struct {
	Season string  `json:"Season,omitempty"`
	Events []Event `json:"Events,omitempty"`
}

File: f1-discord-bot/text-beautifier/calendar.js
const table = require('table')


function calendarToStringTable(calendarInfo){

    let taleConfig = {
        border : table.getBorderCharacters('void'),
        drawHorizontalLine: (index, size) => {
            return index === 0 || index === 1 || index === size
        }
    }
    let elements = [
        ['Dia', 'Circuito']
    ]

    calendarInfo.events.forEach(event => {
        elements.push([event.day, event.circuit])
    })

    return '`' + table.table(elements, taleConfig) + '`'
}

module.exports = {
    calendarToStringTable : calendarToStringTable
}